catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -272,6 +295,19 @@ public Object visit(PropertyIsBetween filter, Object extraData) {
 
     @Override
     public Object visit(PropertyIsEqualTo filter, Object extraData) {
+        FilterFactory2 factory = getFactory(extraData);
+
+        if (filter.getExpression1() instanceof Function) {
```

Comment:
This is the main reason I only implemented PropertyIsEqualTo support for functions every visitor would need to be updated to handle function expressions correctly